server/meshmodel/k8svault-controller/relationships/edge_relationship.json

```diff
@@ -0,0 +1,61 @@
+{
+  "apiVersion": "vault.infra.doodle.com/v1beta1",
+  "kind": "Edge",
+  "metadata": {
+    "description": ""
+  "model": {
+    "name": "k8svault-controller",
+    "version": "0.1.2",
+    "displayName": "K8s Vault Controller",
+    "category": {
+      "name": "Security \u0026 Compliance",
+      "metadata": null
+    },
+    "metadata": {}
+  },
+   "subType": "Mount",
+   "rego_query": "edge_binding_relationship",
+     "selectors": [{
+    "deny": {
+      "from": [],
+      "to": []
+    },
+
+     "allow": {
+        "from": [
+          {
+            "kind": "Secret",
+            "model": "kubernetes",
+              "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                [
+                  "name",
+                  "namespace"
+                ]
+              ],
+           }
+          }
+        ],
+
+        "to": [
+          {
+            "kind": "VaultBinding",
+            "model": "k8svault-controller",
+             "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                [
+                  "spec.secret.name",
```

Comment:
This is not going to work. 
Update the path to `["settings" ,"spec", "secret", "name"]`.